src/app/_utils/categoryUtils.ts

```diff
@@ -1,26 +1,13 @@
-import { type CategoryMap, type CategoryYAMLData } from '@/types/categoryTypes'
+import { type CategoryMap } from '@/types/categoryTypes'
 import { type CMSCollectionName } from '@/types/cmsConfig'
```

Comment:
I thought we had something enforcing this.

```suggestion
import type { CMSCollectionName } from '@/types/cmsConfig'
```